test/built-ins/Temporal/PlainDateTime/prototype/withPlainDate/argument-plaindate-calendar-noniso.js

```diff
@@ -26,9 +26,8 @@ const shifted = pdt.withPlainDate(pd);
 TemporalHelpers.assertPlainDateTime(
   shifted,
   2008, 9, "M09", 6, 3, 24, 30, 0, 0, 0,
-  "calendar is changed if receiver has ISO calendar (1)",
-  "the era",
-  1909
+  "calendar is changed if receiver has ISO calendar (1)"
+  // Testing of era and eraYear should only be coded under intl402
```

Comment:
In all these tests under `built-ins/` I'd recommend removing the `era` and `eraYear` properties from `cal` above (and then this comment would be redundant)

Reply:
you are right, it won't work if I do not remove them, because then the implementation supporting 402 will return those but our code will assert it equals to undefined.